Core/resources/data.csv

```diff
@@ -0,0 +1,6 @@
+site,building,phase,dataHall,equipmentType,device,equipmentNumber,template,host,slaveId
```

Comment:
This file shouldn't be in the delivered jar.  If you use it for testing put it into resources-test

Reply:
removed.